interface/wx/stdpaths.h

```diff
@@ -408,10 +408,14 @@ class wxStandardPaths
     virtual wxString GetUserLocalDataDir() const;
 
     /**
-        Return OS specific directory where project shared liraries are
+        Return OS specific directory where project shared liraries are.
+        The function is different from GetPluginsDir() as PlugIns
+        and Shared Libraries are 2 different things (especially on Mac).
+        Plugin is a library that can be used without restartng the applicatioon
+        whereas the library needes to be installed and the application restarted.
```

Comment:
Thanks, this does explain it, but I'd just say
```suggestion

        The function does the same thing as GetPluginsDir() under non-Mac platforms
        but differs from it under Mac, where plugins (shared libraries loaded by the
        application dynamically while it's running) and shared libraries (that the
        application is statically linked with) are stored in different directories.
```

Reply:
@vadz ,
Fixed in the upcoming commit.